mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -539,6 +539,10 @@ class LinearRegressionTrainingSummary private[regression] (
  * Linear regression results evaluated on a dataset.
  *
  * @param predictions predictions outputted by the model's `transform` method.
+ * @param predictionCol Field in "predictions" which gives the predicted value of the label at
+ *                      each instance.
+ * @param labelCol Field in "predictions" which gives the true label of each instance
+ *                 (if available).
```

Comment:
I added "(if available)" to these descriptions, but isn't the `labelCol` for sure going to be defined in any dataset that was trained or evaluated on that will lead to a summary object?